### ISSUE TITLE
Update Controls to WinUI 3 sources

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/uno#14275]
### What would you like to be added

Now that Microsoft has open-sourced WinUI 3, we should align our controls with the latest implementation and add any newly introduced ones.

<table>
  <tr>
    <td>Control</td>
    <td>Status</td>
    <td>Assignee</td>
  </tr>
  <tr>
    <td>AnimatedIcon</td>
    <td>Blocked by Composition API</td>
    <td></td>
  </tr>
  <tr>
    <td>AnimatedVisualPlayer</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>AnnotatedScrollBar</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>AutoSuggestBox</td>
    <td>📆</td>
    <td>@Youssef1313</td>
  </tr>
  <tr>
    <td>Breadcrumb</td>
    <td>✅ 1.5.3 </td>
    <td>@morning4coffe-dev, @MartinZikmund</td>
  </tr>
  <tr>
    <td>ColorPicker</td>
    <td>📆(#14316)</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>ComboBox</td>
    <td>📆</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>ContentDialog</td>
    <td>📆</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>CommandBarFlyout</td>
    <td>📆 (#10411)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>DropDownButton</td>
    <td>✅</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>Expander</td>
    <td>✅</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>IconSource</td>
    <td>✅14330</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>Imagelcon</td>
    <td>📆</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>InfoBadge</td>
    <td>✅</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>InfoBar</td>
    <td>✅ (#14100)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>ItemContainer</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>ItemsView</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>LayoutPanel</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>MenuBar</td>
    <td>📆(#14290)</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>NumberBox</td>
    <td>📆</td>
    <td>@Youssef1313</td>
  </tr>
  <tr>
    <td>PagerControl</td>
    <td>✅</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>ParallaxView</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>PersonPicture</td>
    <td>✅</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>PipsPager</td>
    <td>✅</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>ProgressBar</td>
    <td>📆(low priority for now)</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>ProgressRing</td>
    <td>📆(low priority for now)</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>PullToRefresh</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>RadialGradientBrush</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>RadioButtons</td>
    <td>📆(low priority for now)</td>
    <td>@morning4coffe-dev</td>
  </tr>
  <tr>
    <td>RadioMenuFlyoutItem</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>RatingControl</td>
    <td>✅ (#14104)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>Repeater</td>
    <td>📅</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>ScrollPresenter</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>ScrollView</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>SplitButton</td>
    <td>✅(#14440)</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>SplitView</td>
    <td>📅</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>StandardUICommand</td>
    <td>✅ (#14099)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>SwipeControl</td>
    <td>Blocked by Composition API</td>
    <td></td>
  </tr>
  <tr>
    <td>TabView</td>
    <td>📆</td>
    <td>@ramezgerges</td>
  </tr>
  <tr>
    <td>TeachingTip</td>
    <td>✅ </td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>TextCommandBarFlyout</td>
    <td>📆 (#10411)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>TitleBar</td>
    <td></td>
    <td></td>
  </tr>
  <tr>
    <td>TimePicker</td>
    <td>📆 (#14097)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>TreeView</td>
    <td>✅ (#14103)</td>
    <td>@MartinZikmund</td>
  </tr>
  <tr>
    <td>TwoPaneView</td>
    <td>✅ (#14102)</td>
    <td>@MartinZikmund</td>
  </tr>
</table>

✅ - Done
📆 - In progress

**Comment by MartinZikmund** [comment]
Closing in favor of separate existing issues
